tests/D2L.CodeStyle.Analyzers.Test/Specs/BlockingAnalyzer.cs

```diff
@@ -0,0 +1,61 @@
+﻿// analyzer: D2L.CodeStyle.Analyzers.Async.BlockingAnalyzer

```

Comment:
Adding test cases for methods within methods.

```
public static void Foo() {
	static void Bar() { }
	Bar();
}
```

Reply:
Done